fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/DateTimeSortRequestHandler.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.handlers;
+
+import com.yahoo.bard.webservice.data.metric.mappers.DateTimeSortMapper;
+import com.yahoo.bard.webservice.druid.model.query.DruidAggregationQuery;
+import com.yahoo.bard.webservice.web.DataApiRequest;
+import com.yahoo.bard.webservice.web.responseprocessors.MappingResponseProcessor;
+import com.yahoo.bard.webservice.web.responseprocessors.ResponseProcessor;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Injects the result set mapper for dateTime based sort if an API request asks for result in descending order.
+ * The time sort mapper should be second last mappers to execute on the result set.
```

Comment:
nvm, I understand now. It's the static ordering of the mappers (as set in the Workflow), _not_ the mapper. My bad 😄